actions/contacts/unconfirm.go

```diff
@@ -41,3 +41,20 @@ func (a *Action) unconfirm(c *gin.Context) {
 	}
 	c.Status(http.StatusOK)
 }
+
+// unconfirmContact will unconfirm contact request
+// Unconfirm contact godoc
```

Comment:
What does `"godoc"` mean in this context

Reply:
It's pretty useless in this context. Removed.
It should mark godoc comment, but You know that.